frontend/ui/widget/inputtext.lua

```diff
@@ -584,6 +586,17 @@ function InputText:focus()
     self._frame_textwidget.color = Blitbuffer.COLOR_BLACK
 end
 
+-- NOTE: This key_map is to be used by kindles with keyboards, it is loosely 'inspired' by the symbol layer on the virtual keyboard but,
```

Comment:
My suspicion meter exploded from something so device-specific but it's secretly generic enough.
```suggestion
-- NOTE: This key_map can be used for keyboards without numeric keys, such as on Kindles with keyboards. It is loosely 'inspired' by the symbol layer on the virtual keyboard but,
```